### DISCUSSION TITLE
Keep getting rejected - please help

### DISCUSSION THREAD

**Discussion by LaneGeek** [community/community#138590] [GitHub Education]
### Hi everyone,

I am a 100% bonafide full time student at an approved college, yet I just keep getting rejected for the Education Developer Pack. I have uploaded transcripts, student ID, you name it.

Please help!

**Comment by aivo9** [comment]
Captain here!

I had the same issue, but I just got approved.

Got rejected multiple times. Time after time. Made few perceptions about how it works, and finally succeeded!

1. Put everything in order in your account settings. Just as the instructions advice you to do. Name, emails, bios, 2 phase authentication, etc.
2. Try to photograph different kind of official school documents with a bad webcam and get rejected multiple times over and over again.
3. Get frustrated.
4. Realize that there is no human approving your entries but artificial intelligence instead "reading" words from the pictures you send.
5. Improvise, adapt, overcome.
6. Open the notebook of your mobile phone and type:

[Your University of Fancy Science]

Student: [insert your name here]
School: [insert your school name (same as in the title) here]
Education: [insert the name of your education here] (I had to add ",Virtual learning" to it cuz I'm not on campus right now, and it got mad)
Study right: [insert the day when you got your study right and the day it ends (in US format e.g. 1 January 2024 - 31 December 2028)]

7. So basically everything is true and you are not lying, but you just made the proof yourself on your mobile phone's notepad.
8. Take a screenshot. Edit the picture with your mobile phone and cut off all the excess space so that the text looks as big as possible in the screen.
9. Open the edited picture on your mobile phone and take the phone so close to your laptop's webcam so that all the words are clearly visible in the picture.
10. Take the photo and check that all the information is visible in the photo.
11. Submit your application.
12. Get approved few minutes later.
13. Not your fault the system of approval is so horrible and broken so feel no shame about cutting the corners.
14. Return the favor and go help others.
15. Enjoy!

Welcome to the community!

*flies away*

**Comment by LaneGeek** [comment]
Thanks, but I just got approved after my 4th try! Now I have to wait for the "pack". I wonder what's in it? Surely at the very least a top of the line latest Surface Pro, along with some MFST stock options, no? After all, this is the hardest ever verification, so the prize must be worth it!!!
